test/integration/targets/nxos_overlay_global/tests/common/sanity.yaml

```diff
@@ -93,13 +86,12 @@
 
   always:
   - name: "Apply N7K specific cleanup config"
-    include: targets/nxos_overlay_global/tasks/platform/n7k/cleanup.yaml
+    include: tasks/platform/n7k/cleanup.yaml
```

Comment:
@nkshrishail Please make sure that the test still runs when the main playbook is in the test/integration folder and not moved outside the directory since that is how we run it in our CI environment.

Reply:
Yes, works from test/integration as well.